### PR TITLE
Minor HTML fixes, enhancements from SteveJ's commit

### PR DESCRIPTION
**Pull request by Toakan**
Signed-off-by: Tom Daykin <tdaykin@live.co.uk>

**By submitting this pull request, I confirm the following:** 

`{Please ensure that your pull request is for the 'devel' branch!}`

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/AdminLTE/blob/master/CONTRIBUTING.md), as well as this entire template.
- [X] I have made only one major change in my proposed changes.
- [X] I have commented my proposed changes within the code.
- [X] I have tested my proposed changes.
- [X] I am willing to help maintain this change if there are issues with it later.
- [X] I give this submission freely and claim no ownership.
- [X] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
- [X] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
- [X] I have Signed Off all commits.

---

**What does this PR aim to accomplish?:**

I've pulled the minor HTML fixes from /pull/704 in order to get them commited. The other enhancements have been left alone, pending confirmation from Stevej about the state of his 

**What documentation changes (if any) are needed to support this PR?:**
None

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -82,7 +82,7 @@ function getinterval()
     <div class="col-md-12">
     <div class="box" id="queries-over-time">
         <div class="box-header with-border">
-          <h3 class="box-title">Queries over last <?php echo getinterval(); ?> hours</h3>
+          <h3 class="box-title">Queries (Last 24 Hours)<?php echo getinterval(); ?> hours</h3>
```
Comment (Mcat12) [inline index.php]: This would expand to `Queries (Last 24 Hours)24 hours`

File: index.php [hunk 2]
```diff
@@ -107,7 +107,7 @@ function getinterval()
     <div class="col-md-12">
     <div class="box" id="clients">
         <div class="box-header with-border">
-          <h3 class="box-title">Clients (over time)</h3>
+          <h3 class="box-title">Clients (Last 24 Hours)</h3>
```
Comment (Mcat12) [inline index.php]: If this includes the interval, it should use `getinterval()`

File: scripts/pi-hole/php/header.php [hunk 3]
```diff
@@ -452,7 +452,7 @@ function pidofFTL()
                 <!-- Whitelist -->
                 <li<?php if($scriptname === "whitelist"){ ?> class="active"<?php } ?>>
                     <a href="list.php?l=white">
-                        <i class="fa fa-check-circle-o "></i> <span>Whitelist</span>
+                        <i class="fa fa-pencil-square-o"></i> <span>Whitelist</span>
```
Comment (Mcat12) [inline scripts/pi-hole/php/header.php]: This was purposely set to `fa-check-circle-o` from `fa-pencil-square-o`, so it should not be changed back.

File: settings.php [hunk 4]
```diff
@@ -378,172 +378,171 @@
...
-                        <div class="row">
-                            <?php
-                            $dhcp_leases = array();
-                            if ($DHCP) {
-                                // Read leases file
-                                $leasesfile = true;
-                                $dhcpleases = @fopen('/etc/pihole/dhcp.leases', 'r');
-                                if (!is_resource($dhcpleases))
-                                    $leasesfile = false;
+							      ?>><strong>DHCP server enabled</strong></label>
```
Comment (Mcat12) [inline settings.php]: Keep indentation using spaces, not tabs.

**Comment by Mcat12** [comment]
@Toakan have you seen the requested changes?

**Comment by Toakan** [comment]
No I hadn't. Will review these and get back.

**Comment by Toakan** [comment]
Closing as these changes would be better done moving all styling to css rather than inline.

**Pull request closed without merging**